tests/fixtures/config/sample-config.php

```diff
@@ -145,4 +145,27 @@
         ],
 
     ],
-];
+
+    // 'key'=>env('KEY'')
```

Comment:
Can you make some of these tests test for double quotes instead of just all single quotes?

Reply:
yes, i add later